### ISSUE TITLE
Metal Gear Solid V: The Phantom Pain - BLES02102 - Sometimes the rpcs3 freezing at the end of 'Building SPU cache'

### ISSUE THREAD

**Issue by visious25** [RPCS3/rpcs3#15533]
### Quick summary

Hi, sometimes the rpcs3 freezing at the end of 'Building SPU cache'. Also the graphics looks horrible especially at night., the screen in game at night getting green.

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/15153018/RPCS3.log)

### System configuration

OS: Arch Linux x86_64
Kernel: 6.8.7-zen1-2-zen
CPU: Intel i7-8750H
GPU: NVIDIA GeForce GTX 1050 Ti
Nvidia driver: 550.78

**Comment by AniLeo** [comment]
Any graphical issues should not go in the same ticket as SPU compilation failure.

There are several settings in the log that are not defaults nor the settings in the RPCS3 Wiki page for the game without any explanation to why they're needed for the game to work. **Needs retest with only the required settings.**

Another thing to note that could be related: this is a rpcs3-git build, which is fine as it's supported, but results could differ in the AppImage.

**Comment by visious25** [comment]
> Any graphical issues should not go in the same ticket as SPU compilation failure.

SPU compilation failure.is really annoying.

> There are several settings in the log that are not defaults nor the settings in the RPCS3 Wiki page for the game without any explanation to why they're needed for the game to work. **Needs retest with only the required settings.**

Basically I'm testting the settings in the RPCS3, I'm not sure which are the best settings for this game

> Another thing to note that could be related: this is a rpcs3-git build, which is fine as it's supported, but results could differ in the AppImage.

If you want I can use AppImage, it would be easier to you to get the results.

If there something than I can do for you, feel free and please tell me.

**Comment by Abuseware** [comment]
I have exact same issue running with stock config on ArchLinux, no mater what version I'm using (AppImage, Flatpak, AUR), game looks up for pathes like `/dev_bdvd/PS3_GAME/USRDIR/dev_bdvd/PS3_GAME/USRDIR/` which are malformed imo. Tried with game in versions from 1.0 to 1.11, nothing changes.

**Comment by visious25** [comment]
To avoid green screen in dark places, leave this setting to default:
`Configuration > GPU > Resolution Scale (Disable Strict Mode) > 100% (1280x720) (Default)` 
That worked for me.

**Comment by requeijaum** [comment]
Debian testing here with rpcs3 running as AppImage and PSN US updated version of the game.

PC is Ryzen 5600G with integrated graphics (Vulkan-capable, Vega 7) running in amdgpu kernel driver.

I just can't boot this game and I don't know why. Tried multiple things already.

**Comment by digant73** [comment]
is that still an issue?

**Comment by visious25** [comment]
@digant73 Yes it's still the same issue. Sometimes failing to boot the game.

**Comment by DTechGuy** [comment]
resurrecting this issue, Latest build still has the issue on Linux. Using the Latest appimage. rpcs3-v0.0.41-19456-2ce45bd9_linux64. Game further crashes if it makes it past Building the cache.
